diagnostic.md

```diff
@@ -5,74 +5,90 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+The purpose of a backend is to allow multiple users access to an API to connect them to a single application.
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```md
-// your response here
+The model
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
 ```md
-// your response here
+The controller
 ```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```md
-// your response here
+Because the HTML and CSS we create will do everything the views are intended to do.
 ```
 
 What does C.R.U.D stand for?
 
 ```md
-// your response here
+Create Read Update Destroy
 ```
 
 In which part of the MVC pattern can we find C.R.U.D actions?
 
 ```md
-// your response here
+In the controller
 ```
 
 List at least 5 standard rails actions that C.R.U.D requests correspond to?
 
 ```md
-// your response here
+index
+new
+create
+show
+update
 ```
 
 A user action fires a `GET` request for `/people/1`. Explain in detail each step
 required for data to be returned to the client. (bullet points or ordered list)
 
 ```md
-// your response here
```

Comment:
- The server receives the GET requests and uses routes to find out which controller to use.
- The web server then uses the dispatcher to create a new controller, call the action and pass the parameters.
- Controllers parse user requests and tell the model to perform the request.
- The model talks to the database, stores and validates data, and performs the business logic.
- The model returns a response to the controller.
- The controller returns the response body and metadata to the server.
- The server combines the raw data into an HTTP response and sends it to the user.